.travis.yml

```diff
@@ -17,21 +17,22 @@ matrix:
       env:
         - CS_CHECK=true
     - php: 7.1
-    - php: hhvm 
+    - php: hhvm
   allow_failures:
     - php: hhvm
 
 before_install:
   - if [[ $TEST_COVERAGE != 'true' ]]; then phpenv config-rm xdebug.ini || return 0 ; fi
   - composer self-update
   - if [[ $TEST_COVERAGE == 'true' ]]; then composer require --dev --no-update satooshi/php-coveralls:^1.0 ; fi
+  - if [[ $TEST_COVERAGE == 'true' ]]; then composer require --dev --no-update michaelmoussa/php-coverage-checker:^1.1.0 ; fi
```

Comment:
It marks the build as failed if test coverage drops below the specified value. Coveralls displays the current coverage percentage and links to the HTML output, but AFAIK there isn't a way to tell it "if the coverage is below X, fail the build".

If you'd prefer not to enforce that, I can remove it, but it'd be nice to ensure component repo coverage stays at 100% now that it has reached that point.